include/osgAnimation/MorphGeometry

```diff
@@ -59,7 +60,10 @@ namespace osgAnimation
         virtual const char* libraryName() const { return "osgAnimation"; }
         virtual const char* className() const { return "MorphGeometry"; }
 
-        virtual void transformSoftwareMethod();
```

Comment:
Removing this function would break my code actually (I call transform manually from a cull visitor, so there's no overhead for non-visible objects). Can we re-add this function (just make it call the MorphTransformImplementation internally)?

Reply:
Hm... I guess calling it 'transformSoftwareMethod' wouldn't make a lot of sense then, if the implementation could be something else. So if it's renamed, I would still need to adjust my code. Is there any reason to merge this change now, when software is still the only available method? In other words, can we hold back this change until a HW implementation is added, so that the API doesn't unnecessarily break in the meantime?

Reply:
you could call `rig.getMorphTransformImplementation()()` instead
but i see your point